loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/io/ValidationWriterFactory.java

```diff
@@ -19,4 +20,7 @@
 
     ValidationWriter create(String id, Class<? extends TableFormatterFactory> formatterFactoryClass, Writer writer, boolean verbose, ValidationType validationType, boolean compareResults);
 
+    default ValidationWriter create(String id, Class<? extends TableFormatterFactory> formatterFactoryClass, TableFormatterConfig formatterConfig, Writer writer, boolean verbose, ValidationType validationType, boolean compareResults) {
```

Comment:
Add @deprecated and a default body to the previous method. Fix all the implementations. I’m pretty sure that this code is not used outside this module. You can probably break the API here.

Reply:
Done